lib/spack/spack/build_systems/nmake.py

```diff
@@ -30,73 +30,119 @@ class NMakePackage(spack.package_base.PackageBase):
 @spack.builder.builder("nmake")
 class NMakeBuilder(BaseBuilder):
     """The NMake builder encodes the most common way of building software with
-    NMake on Windows. It has three phases that can be overridden, if need be:
+    Mircosoft's MSBuild tool. It has two phases that can be overridden, if need be:
```

Comment:
Both this and `MSBuilder` claim to encode the most common way of building software "with Microsoft's MSBuild tool": I presume this part was copy/pasted and could use an adjustment.